service/Abstractions/MemoryStorage/IBatchMemoryDb.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Microsoft.KernelMemory.MemoryStorage;
+
+/// <summary>
+/// Common interface MemoryDB that supports batching operations.
+/// </summary>
+public interface IBatchMemoryDb : IMemoryDb
+{
+    /// <summary>
```

Comment:
pls fix the docs below, using plural etc.  Also you can remove "creates the given index if it does not exist." here and in `IMemoryDb` because that's not how we're implementing these methods, which throw IndexNotFound instead

Reply:
done in https://github.com/microsoft/kernel-memory/pull/441